From 24a8c279da98fe7e586a9fc3cd91175c9e60ffef Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ingo=20Kl=C3=B6cker?= <dev@ingo-kloecker.de>
Date: Mon, 21 Aug 2023 21:32:40 +0200
Subject: build: Suggest out-of-source build

* autogen.rc (final_info): Suggest to run configure from a build
subdirectory.
--

We recommend out-of-source builds. The suggestion what to do next shown
by autogen.sh should reflect this.

GnuPG-bug-id: 6673

diff --git a/autogen.rc b/autogen.rc
index 1759ef93..6d6ea93f 100644
--- a/autogen.rc
+++ b/autogen.rc
@@ -11,4 +11,4 @@ case "$myhost" in
 esac
 
 
-final_info="./configure --enable-maintainer-mode && make"
+final_info="mkdir build && cd build && ../configure --enable-maintainer-mode && make"
-- 
2.41.0

